cmd/test/main.go

```diff
@@ -219,6 +222,19 @@ func New(ctx *TestCommandContext) *cobra.Command {
 	return testCommand
 }
 
+func (flags *TestCommandFlags) ToMapping() map[string]interface{} {
+	val := reflect.Indirect(reflect.ValueOf(flags))
```

Comment:
https://stackoverflow.com/questions/18926303/iterate-through-the-fields-of-a-struct-in-go#:~:text=19,a%20new%20option

Reply:
this is only a static reflection of the object type. there's no way to get the value of a field without using `valueOf` that makes the code look more complicated than it was